python/desc/imsim/imSim.py

```diff
@@ -883,7 +886,7 @@ def make_psf(psf_name, obs_md, log_level='WARN', rng=None, **kwds):
                              logger=logger, **kwds)
     return psf
 
-def save_psf(psf, outfile):
+def save_psf(psf, outfile, screen_file=None):
```

Comment:
I was kind of hoping that we could get a picklable object from galsim containing the phase screen data and retain the `save_psf` and `load_psf` interface so the user doesn't have to keep track of two files for the atmospheric PSF, i.e., do something along the lines of 
```
def save_psf(psf, outfile, screen_file=None):
    """
    Save the psf as a pickle file.
    """
    # Set any logger attribute to None since loggers cannot be persisted.
    if hasattr(psf, 'logger'):
        psf.logger = None

    with open(outfile, 'wb') as output:
        if isinstance(psf, AtmosphericPSF):
            phase_screens = galsim.phase_screens.get_picklable()
            pickle.dump((psf, phase_screens), output)
        else:
            pickle.dump(psf, output)


def load_psf(psf_file, log_level='INFO'):
    """
    Load a psf from a pickle file.
    """
    with open(psf_file, 'rb') as fd:
        psf_data = pickle.load(fd)

    if len(psf_data) == 2:
        psf = psf_data[0]
        galsim.phase_screens.set_data(psf_data[1])
    else:
        psf = psf_data

    # Since save_psf sets any logger attribute to None, restore
    # it here.
    if hasattr(psf, 'logger'):
        psf.logger = get_logger(log_level, 'psf')

    return psf
```